src/Opprintast.ml

```diff
@@ -11,7 +11,7 @@
 (*                                                                        *)
 (*   All rights reserved.  This file is distributed under the terms of    *)
 (*   the GNU Lesser General Public License version 2.1, with the          *)
-(*   special exception on linking described in the file LICENSE.          *)
+(*   special exception on linking described in the file LICENSW.          *)
```

Comment:
erroneous change?

Reply:
Ah yes, my `sed` was a bit too brutal. Thanks for catching this!